add limitation (#119)
cybozu-go/topolvm: docs/limitations.md

-------------------------

TopoLVM does not recognize `mountOptions` of `StorageClass` currently.

Capacity-aware scheduling may go wrong
-------------------------

Node storage capacity annotation is not updated in TopoLVM's extended scheduler.

> Therefore, when multiple pods requesting TopoLVM volumes are requested at once, the extended scheduler cannot reference the exact capacity of the underlying LVM volume group.

Reviewer: ```suggestion
Therefore, when multiple pods requesting TopoLVM volumes are created at once, the extended scheduler cannot reference the exact capacity of the underlying LVM volume group.
```

-------------------------

TopoLVM does not recognize `mountOptions` of `StorageClass` currently.

> Time-lag in updating node capacity 

Reviewer: ```suggestion
Capacity-aware scheduling may go wrong
```

-------------------------

> Node capacity is not updated in TopoLVM's extended scheduler.

Reviewer: What capacity?  Storage?

> Therefore, if many volumes are requested at the same time, they may be assigned to one node and cannot be created due to the shortage of node storage.

Reviewer: ```suggestion
Therefore, when multiple pods requesting TopoLVM volumes are requested at once, the extended scheduler cannot reference the exact capacity of the underlying LVM volume group.
```
